generator/integration_tests/golden/v1/golden_thing_admin_client.h

```diff
@@ -405,6 +419,12 @@ class GoldenThingAdminClient {
   future<StatusOr<google::test::admin::database::v1::UpdateDatabaseDdlMetadata>>
   UpdateDatabaseDdl(google::test::admin::database::v1::UpdateDatabaseDdlRequest const& request, Options opts = {});
 
+  StatusOr<google::longrunning::Operation>
```

Comment:
We need to document the client methods.

And the documentation should probably mention that the `NoAwaitTag` overload is for advanced use cases.